### PR TITLE
Ohio

### PR DESCRIPTION
**Pull request by EducatedSuddenBucket**
Ohio

### CODE REVIEW

**Comment by orxngc** [comment]
i don't see the point of this

File: src/docs/render/index.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Create a JSON file inside the `domains` directory (`domains/<subdomain>.json`) w
     }
 }
 ```
-**Note:** In the owner section, you can add any social media handle, such as Discord. If you add another social media account, you can omit the email and Twitter fields. However, the GitHub username is mandatory.
+**Note:** In the owner section, you can add any social media handle, such as Discord. If you add another social media account, you can omit the email and Twitter fields. However, the GitHub username is mandatory.In your pull request dont forgot to provide a preview of the site.
```
Comment (orxngc) [inline src/docs/render/index.md:26]:
```suggestion
**Note:** In the owner section, you can add any social media handle, such as Discord. If you add another social media account, you can omit the email and Twitter fields. However, the GitHub username is mandatory. In your pull request don't forgot to provide a preview of your site.
```

File: src/docs/netlify/index.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ Create a JSON file inside the `domains` directory (`domains/<subdomain>.json`) w
     }
 }
 ```
-**Note:** In the owner section, you can add any social media handle, such as Discord. If you add another social media account, you can omit the email and Twitter fields. However, the GitHub username is mandatory.
+**Note:** In the owner section, you can add any social media handle, such as Discord. If you add another social media account, you can omit the email and Twitter fields. However, the GitHub username is mandatory.In your pull request dont forgot to provide a preview of the site.
```
Comment (orxngc) [inline src/docs/netlify/index.md:26]:
```suggestion
**Note:** In the owner section, you can add any social media handle, such as Discord. If you add another social media account, you can omit the email and Twitter fields. However, the GitHub username is mandatory. In your pull request don't forgot to provide a preview of your site.
```

File: src/docs/index.md [hunk 3]
```diff
@@ -33,7 +33,7 @@ These are the subdomains run by the is-a.dev staff team. Do not trust any subdom
 - register.is-a.dev (old registration site, redirects to repo)
 - team.is-a.dev (team documentation)
 - www.is-a.dev (main website + docs)
-
+- check.is-a.dev (check if a domain under is.-a.dev is available)
```
Comment (orxngc) [inline src/docs/index.md:36]:
```suggestion
```
this is not official

File: src/docs/index.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ You can read about the domains JSON file structure [here](domain-structure/index
 - [Other Services](other/index.md)
 
 ## Official Subdomains
-These are the subdomains run by the is-a.dev staff team. Do not trust any subdomain that is not on this list that claims to be us.
+These are the subdomains run by the ~~is-a.dev staff team.~~ Do not trust any subdomain that is not on this list that claims to be us.
```
Comment (orxngc) [inline src/docs/index.md:19]:
```suggestion
These are the subdomains run by the is-a.dev staff team. Do not trust any subdomain that is not on this list that claims to be us.
```
no, why did you do this

File: src/docs/dbh/index.md [hunk 5]
```diff
@@ -34,7 +34,7 @@ Create a JSON file inside `domains` directory (`domains/<subdomain>.json`) with
     }
 } 
 ```
-**Note:** In the owner section, you can add any social media handle, such as Discord. If you add another social media account, you can omit the email and Twitter fields. However, the GitHub username is mandatory.
+**Note:** In the owner section, you can add any social media handle, such as Discord. If you add another social media account, you can omit the email and Twitter fields. However, the GitHub username is mandatory.In your pr rquest dont forgot to provide a preview of the site.
```
Comment (orxngc) [inline src/docs/dbh/index.md:37]:
```suggestion
**Note:** In the owner section, you can add any social media handle, such as Discord. If you add another social media account, you can omit the email and Twitter fields. However, the GitHub username is mandatory. In your pull request don't forgot to provide a preview of your site.
```

**Review by orxngc — CHANGES_REQUESTED** [review]

File: src/docs/dbh/index.md (hunk 5 above)
Comment (wdhdev) [reply src/docs/dbh/index.md:37]: I've never seen such bad English lol

**Review by orxngc — APPROVED** [review]

**Pull request merged**